### DISCUSSION TITLE
希望能支持Waline数据迁移

### DISCUSSION THREAD

**Discussion by dyxang** [walinejs/waline#653] [Q&A] [answered]
**虽然该讨论没有回复，但此问题已经在 #655 得到开发者解决**

我原本是直接Valine 直接迁移到Waline的，我现在尝试从Leancloud迁出到如elephantsql这样的网站。
于是我尝试了两种方法：

1. 直接使用Leancloud里面的数据存储> 数据导出 导出为jsonline> json > csv 然后尝试导入 postgresql
2. Leancloud 结构化数据 >全选class为comment的所有查询结果 >查询结果保存为 csv

两者都会得到`ERROR:  extra data after last expected column`的报错。请问就现阶段有没有可能方便地把Waline数据从Leancloud迁移到别的地方，比如Supabase或elephant的这种地方……其实这折腾的初衷就是觉得Leancloud国际版太慢了（账号不实名网站不备案），看elephant这种有香港的服务器……

**Comment by soxft** [comment]
直接在 waline的后台 导出数据
然后去 其他平台 搭建, 之后在waline后台导入

**Reply by lizheming** [reply]
就是在这个 discussion 之后加了 Waline 的导入导出功能

**Reply by dyxang** [reply]
我了解，以防万一修改了一下内容防止搜到这个讨论的人给忽略了

**Reply by soxft** [reply]
嗯嗯 好的吧

**Comment by Mister-Hope** [comment, marked as answer]
Solved in #655
